charts/observation-service/templates/service-monitor.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.observationService.monitoring.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "observation-service.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{ include "observation-service.labels" . | nindent 4 }}
```

Comment:
i think `nindent` is not needed here

Reply:
It's required for proper indentation of labels, I've checked that.

Reply:
I tested this, and `nindent` is required 😅 I'm merging 🙏🏻